android/app/src/main/java/com/buttercup/MainActivity.java

```diff
@@ -69,7 +69,7 @@ public void checkPerms() {
     public void onActivityResult(int requestCode, int resultCode, Intent data) {
         super.onActivityResult(requestCode, resultCode, data);
         if (requestCode == OVERLAY_PERMISSION_REQ_CODE) {
-            checkPerms();
+            // checkPerms();
```

Comment:
this.

Reply:
Thanks @Thomazella - I can definitely take to testing that on Android :)